PowerShellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingServiceUtilities.cs

```diff
@@ -403,7 +403,9 @@ private void EnterCredentialedRemoteSession(PowerShell powerShell, string remote
                 return;
             }
 
-            string port = remoteName.Split(':').ElementAtOrDefault(1);
+            string[] addressParts = remoteName.Split(':');

```

Comment:
Working on making sure this is fully compatible with IPv4, IPv6, and other possibilities. As for handling stuff correctly, yes. In the end, everything is handled gracefully. Worst case is that we end up giving something invalid to PowerShell and it can't connect which we catch.